### PR TITLE
add Append() method for adding multiple elements to set

### LINKED ISSUE #92: What about adding a `set.Append(vals...T)` method to add multiple values to an existing set?
We've got `NewSet(vals...T)` for creating a set from multiple values.

But today I found myself wanting to add a slice of objects to an existing set... today I have to write a for loop to range over the objects.

What about adding an `append(vals... T)` method... similar to Python, which allows adding multiple objects.

Alternatively `Add(val T) bool` could be modified to take multiple args, which would keep the API surface small. But it's a little undefined that it returns a bool... for multiple objects would that return true if all the objects got added to the existing set, or if at least one object got added?

Thoughts?

### PR DESCRIPTION
**Pull request by SignorMercurio**
This PR implements `Append` method and resolves #92 .

Test results:
```shell
$ go test -cover
PASS
coverage: 94.8% of statements
ok      github.com/deckarep/golang-set/v2       0.767s
```

### CODE REVIEW

**Comment by paullesiak** [comment]
I just switched over to mapset from using a home grown set implementation, and really enjoying mapset. This functionality is the only aspect missing. @deckarep any chance this may be looked at for inclusion?

**Comment by deckarep** [comment]
I should have some time to look at it later today.

Thanks for the interest!

**Comment by deckarep** [comment]
@SignorMercurio  - Hello, looks like this PR has merge conflicts after I merged a few PRs. Do you have any bandwidth to tackle them? I'm still swamped currently but will get to it eventually. I appreciate your contribution!

**Comment by SignorMercurio** [comment]
@deckarep Sure, I've resolved conflicts now.

**Comment by deckarep** [comment]
@SignorMercurio - hmm still seeing conflicts unfortunately. I am unable to merge.

**Comment by SignorMercurio** [comment]
@deckarep  I'm not sure why I cannot see the conflicts now:

image

Is there anything I can do to help?

**Comment by danielchalef** [comment]
@deckarep I'd love to avoid for loops for adding multiple values to a set. Would you have time soon to merge this PR?

File: set.go [hunk 1]
```diff
@@ -43,6 +43,10 @@ type Set[T comparable] interface {
 	// the item was added.
 	Add(val T) bool
 
+	// Append multiple elements to the set. Returns
```
Comment (deckarep) [inline set.go:46]: LGTM

**Review by deckarep — COMMENTED** [review]
LGTM

**Review by deckarep — APPROVED** [review]

**Pull request merged**